backend/package.json

```diff
@@ -14,9 +14,17 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+<<<<<<< HEAD
```

Comment:
This commit has merge conflicts messages not removed which is a serious issue.
Kindly resolve the conflicts and remove the conflict messages.

Reply:
I don't know hot to resolve them. I have tried but they appear again and again. Please help.